### PR TITLE
Allow for global workspace file

### PR DESCRIPTION
**Pull request by eyalk11**
The changes in handleAutochdir are by chatgpt you can remove them. 

It is not that hard.

### CODE REVIEW

**Comment by Konfekt** [comment]
Sounds useful. Could you add documentation?

File: autoload/ctrlspace/util.vim [hunk 1]
```diff
@@ -56,17 +56,24 @@ function! s:handleSwitchbuf(switch) abort
...
+        set noautochdir
+    elseif exists("s:acdWasOn") && s:acdWasOn
+        set autochdir
         unlet s:acdWasOn
     endif
 endfunction
 
 function! ctrlspace#util#WorkspaceFile() abort
-    return s:internalFilePath("cs_workspaces")
+    if exists('g:ctrlspaceWorkspace')
```
Comment (Konfekt) [inline autoload/ctrlspace/util.vim:71]: This variable seems undocumented. All other use Pascal case; how come this camel case?

Comment (eyalk11) [reply autoload/ctrlspace/util.vim:71]: I added it. You can chenge the name

Comment (Konfekt) [reply autoload/ctrlspace/util.vim:71]: Would you mind using Pascal case and documenting the variable?

**Comment by eyalk11** [comment]
I did so.

**Comment by Konfekt** [comment]
Great, please go ahead and push your amendments

**Comment by eyalk11** [comment]
I pushed. It was git push --force.

**Comment by eyalk11** [comment]
Will it be merged?

**Comment by Konfekt** [comment]
Please fix the pending comments in https://github.com/vim-ctrlspace/vim-ctrlspace/pull/320/files first

**Comment by eyalk11** [comment]
What comments weren't addressed? I fixed documentation and changed variable name.

**Comment by Konfekt** [comment]
Those titled "Pending" (without reply) in [the code](https://github.com/vim-ctrlspace/vim-ctrlspace/pull/320/files)

**Comment by eyalk11** [comment]
Are you hallucinating? The comments were resolved and are not accessible.

**Comment by eyalk11** [comment]
image
Here is the documentation and changed the variable name. I don't get it.

**Comment by Konfekt** [comment]
grafik

This is what your pull request looks like by now

**Comment by Konfekt** [comment]
Maybe an independent contributor (@swoh816 , would you be so kind ?) could clarify if these only show up on my machine?

**Comment by eyalk11** [comment]
I don't see those. Did you write them in the pull request?

File: autoload/ctrlspace/util.vim [hunk 2]
```diff
@@ -66,7 +66,12 @@ function! s:handleAutochdir(switch) abort
 endfunction
 
 function! ctrlspace#util#WorkspaceFile() abort
-    return s:internalFilePath("cs_workspaces")
+    if exists('g:CtrlSpaceWorkspaceFile')
```
Comment (eyalk11) [inline autoload/ctrlspace/util.vim:69]: test comment

**Comment by lisez** [comment]
Maybe you need to submit your reviews?

> **Before you submit your review, your line comments are pending and only visible to you.** You can edit pending comments anytime before you submit your review. To cancel a pending review, including all of its pending comments, click Review changes above the changed code, then click Abandon review.

from: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request

**Review by Konfekt — CHANGES_REQUESTED** [review]

**Comment by Konfekt** [comment]
@lisez , @eyalk11 Can you see the comments now?

**Comment by lisez** [comment]
> @lisez , @eyalk11 Can you see the comments now?

Yes, I can see comments. fyi, I have already seen your comments after Nov 15.

**Comment by Konfekt** [comment]
Let's continue at above-mentioned PR superseding this one grown stale.
Feedback highly appreaciated @lisez and @eyalk11

**Pull request closed without merging**